src/front-end/typescript/lib/http/api/proposal/code-with-us.ts

```diff
@@ -20,15 +20,27 @@ export function create<Msg>(): crud.CreateAction<
   return crud.makeCreateAction(NAMESPACE, rawCWUProposalToCWUProposal);
 }
 
+/**
+ * Parses URL parameters prior to creating a read request for many CWU proposals
+ *
+ * @param opportunityId
+ * @param orgProposals
+ */
 export function readMany<Msg>(
-  opportunityId?: Id
+  opportunityId?: Id,
+  orgProposals?: Id
 ): crud.ReadManyAction<Resource.CWUProposalSlim, string[], Msg> {
+  const params = new URLSearchParams({
+    opportunity:
+      opportunityId !== undefined
+        ? window.encodeURIComponent(opportunityId)
+        : "",
+    organizationProposals: orgProposals !== undefined ? orgProposals : ""
```

Comment:
Do we need to encode the URI component for orgProposals (same applies for SWU and TWU?)